aws/provider.go

```diff
@@ -545,6 +545,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_secretsmanager_secret_version":                resourceAwsSecretsManagerSecretVersion(),
 			"aws_ses_active_receipt_rule_set":                  resourceAwsSesActiveReceiptRuleSet(),
 			"aws_ses_domain_identity":                          resourceAwsSesDomainIdentity(),
+			"aws_ses_domain_identity_policy":                   resourceAwsSesDomainIdentityPolicy(),
```

Comment:
Two larger items:
* Given that this resource can work with both SES Identity Policies for both domains and emails, we should consider naming it `aws_ses_identity_policy` as there would be no benefit to having separate resources between the two. The file, resource, and resource testing function names will need to be updated to reflect this change.
* This new resource is missing resource documentation in a new `website/docs/r/ses_identity_policy.html.markdown` file and a website sidebar link in `website/aws.erb`